airflow/models.py

```diff
@@ -1006,6 +1007,7 @@ def mark_success_url(self):
         iso = self.execution_date.isoformat()
         BASE_URL = configuration.get('webserver', 'BASE_URL')
         return BASE_URL + (
+            configuration.get_url_prefix() +
```

Comment:
Should this be something using `posixpath.join` in case of trailing slashes ?

Reply:
Yes, it requires to aways write a backslash before, so this get url returns either nothing (no prefix) or a path that does not ends with a backslash.
Double slash on a file path is not really a problem, it is more on some web server that might be an issue. But I do not know if I can put a urljoin/ file join everywhere, especially in JavaScript file